docs/client-sdk/javascript/tutorials/filter-and-label-conversations-by-id-and-metadata.md

```diff
@@ -0,0 +1,168 @@
+---
+sidebar_label: Filter and label conversations
+sidebar_position: 3
+---
+
+# Filter and label conversations using conversation IDs and metadata
+
+With XMTP, a conversation is a set of messages sent between two blockchain account addresses (addresses). Two addresses can have multiple ongoing conversations.
+
+You can assign an ID and other metadata to a conversation to use as conversation filters and labels in your app.
+
+For example, you can use conversation IDs and metadata to filter conversations by:
+
+* An app  
+Filter to display only conversations created by an app.
+
+* A message type<!--added this one because it matches the example we show in the code snippets - okay?-->  
+Filter to display only conversations that are direct messages or notifications, for example
+
+* A subject line  
+Filter to display only conversations with a specific subject line, as in email
+
+* An NFT  
+Filter to display only conversations associated with a specific NFT
```

Comment:
This might be over selling what is possible here. We can link a conversation to be "about a NFT", but we don't have any concept of whether someone actually owns a NFT

Reply:
Thank you for this clarification, Nick! Yes, I see what you are saying. I've surfaced this detail - that the filtering/linking is not based on NFT ownership. I'll also follow up with Saul to clarify the use case.